hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableAccountStore.java

```diff
@@ -223,9 +223,9 @@ private HederaAccount getContractLeaf(@NonNull final ContractID id) {
                                         ByteString.copyFrom(evmKeyAliasAddress).toStringUtf8());
                             }
                         }
-                        yield entityNum.num();
+                        yield entityNum == null ? new EntityNumValue().num() : entityNum.num();
```

Comment:
Should this be using `EntityNumValue.DEFAULT_VALUE` (or similar)?  It's somewhat confusing to create a new `EntityNumValue` just to get a default number...

Reply:
yeah agreed. Will add static field